pyproject.toml

```
@@ -12,7 +12,9 @@ dynamic = ["version", "readme"]
 dependencies = [
     "typing-extensions",
     "pint",
-    "pydantic>1,<3"
+    "pydantic>1,<3",
```

```suggestion
    "pydantic>2,<3",
```

This doesn't work with Pydantic 1.x, does it?

---

Good point, I put >=2 now.